modules/gdscript/language_server/gdscript_language_server.h

```diff
@@ -45,7 +45,7 @@ class GDScriptLanguageServer : public EditorPlugin {
 	bool started = false;
 	bool use_thread = false;
 	String host = "127.0.0.1";
-	int port = 6008;
+	int port = 6005;
```

Comment:
I think the commit message and PR should mention that this changes the default GDScriptLangaugeServer port to `6005`, as it might result in plugins using it no longer working by default (given the default port has changed).